frontend/src/components/features/ZakenApiFeatureConfiguration.tsx

```diff
@@ -191,6 +192,24 @@ const ZakenApiFeatureConfiguration = ({
                                                    id="secret"
                                                />
                                            </FormField>
+                                           <FormField
+                                               label={
+                                                   <FormLabel htmlFor={"zaak-types-ids-excluded"}>
+                                                       <FormattedMessage
+                                                           id={"features.zakenapi.zaak-types-ids-excluded"}/>
+                                                   </FormLabel>
+                                               }
+                                               description={
+                                                   <Paragraph>
+                                                       <FormattedMessage
+                                                           id={"features.zakenapi.zaak-types-ids-excluded.description"}/>
+                                                   </Paragraph>
+                                               }>
+                                               <TextInput
+                                                   {...register("properties.zaak-types-ids-excluded")}
+                                                   id="client-id"
```

Comment:
duplicate id, did you mean this?
```suggestion
                                                   id="zaak-types-ids-excluded"
```